scripts/gh/pr-status.sh

```
@@ -24,6 +24,20 @@ fi
 BRANCH=$(git branch --show-current)
 printf "${BLUE}[info] branch: %s${NC}\n" "$BRANCH"
 
+# Check if PR exists and get its draft status
+PR_INFO=$(gh pr view "$BRANCH" --json isDraft,number 2>/dev/null || echo "")
+if [ -n "$PR_INFO" ]; then
+  IS_DRAFT=$(echo "$PR_INFO" | jq -r '.isDraft')
+  PR_NUMBER=$(echo "$PR_INFO" | jq -r '.number')
+  if [ "$IS_DRAFT" = "true" ]; then
+    printf "${YELLOW}[info] PR #%s is in DRAFT mode${NC}\n" "$PR_NUMBER"
+    printf "${DIM}       Before marking ready: run 'make pre-pr' to validate${NC}\n"
```

Shellcheck (0.11.0)

[info] Don't use variables in the printf format string. Use printf '..%s..' "$foo".

(SC2059)